core/src/main/scala/org/apache/spark/ui/jobs/StagePage.scala

```diff
@@ -177,6 +192,10 @@ private[ui] class StagePage(parent: StagesTab) extends WebUIPage("stage") {
           <div class="additional-metrics collapsed">
             <ul>
               <li>
+                <strong>Locality Level Summary: </strong>
```

Comment:
Hm, actually on final thought, is this the right place to show this info? it's dumped into a list of metrics, but all the others are controlled by arrows and checkboxes.